powershell/resources/assets/build-module.ps1

```diff
@@ -86,6 +88,14 @@ if(-not $Debugger) {
   $null = Remove-Item -Recurse -ErrorAction SilentlyContinue -Path (Join-Path $binFolder 'Debug'), (Join-Path $binFolder 'Release')
 }
 
+if ($isAzure -And '${$project.assemblyInfoPath}') {
+  Write-Host -ForegroundColor Green 'Adding license for assemblyinfo ...'
```

Comment:
Let's remove **write-host** here, as we do not do similar thing when adding license header for the other generated code.